.github/workflows/deploy-on-kind.yaml

```diff
@@ -0,0 +1,64 @@
+---
+name: Testing on Kind
+on:
+  push:
+    paths:
+      - 'charts/jenkins/**'
+  pull_request:
+    paths:
+      - 'charts/jenkins/**'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        kubernetes_version:
+          - "kindest/node:v1.20.0"
+          - "kindest/node:v1.19.1"
+          - "kindest/node:v1.18.8"
+          - "kindest/node:v1.17.11"
+          - "kindest/node:v1.16.15"
```

Comment:
Following https://en.wikipedia.org/wiki/Kubernetes it should be sufficient to test 1.18, 1.19 and 1.20

Reply:
It depends on what versions of Kubernetes you want to support? for example, GKE stable is still 1.16, (regular is 1.17) - I'd expect other cloud providers to lag at a similar rate.

Reply:
Azure is supporting back till 1.16 until next month:
https://docs.microsoft.com/en-us/azure/aks/supported-kubernetes-versions#aks-kubernetes-release-calendar

Reply:
Good arguments to support 1.16
Only downside that ci jobs will take longer.

Reply:
the CI jobs are run in parallel so shouldn't add much time to the overall build.

Reply:
Could you integrate the multi cluster matrix in the existing lint-test workflow?